.github/workflows/actions_build.yml

```diff
@@ -87,11 +92,12 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
-      - name: Set up JDK ${{ matrix.java }}
+      - id: setup-jdk-17
+        name: Set up JDK 17
         uses: actions/setup-java@v2
         with:
           distribution: 'temurin'
-          java-version: ${{ matrix.java }}
+          java-version: '17'
```

Comment:
Should remove the declaration and the usage of `matrix.java` in this task as well?

Reply:
Fixed. 😉